daemon/src/capability.rs

```diff
@@ -20,19 +92,41 @@ pub mod http {
                     ("sort", "activity"),
                     ("q", &search),
                     ("site", "stackoverflow"),
+                    ("accepted", "true"),
+                    ("closed", "false"),
                 ];
                 let request = client
                     .get("https://api.stackexchange.com/2.2/search/advanced")
                     .query(&params);
-
                 let response = request.send().await?;
+                let payload: stack_exchange::Response<stack_exchange::Question> = response
+                    .json()
+                    .await
+                    .expect("StackOverflow questions HTTP response");
+                let question_ids: Vec<String> = payload
+                    .items
+                    .iter()
+                    .map(|q| q.question_id.to_string())
+                    .collect();
+                let ids = question_ids.join(";");
+                let url: String = format!(
+                    "https://api.stackexchange.com/2.2/questions/{}/answers",
+                    ids
+                );
+                let params = [("filter", "withbody"), ("site", "stackoverflow")];
 
-                let body = response.bytes().await?;
+                let request = client.get(&url).query(&params);
+                let response = request.send().await?;
+                let answers: stack_exchange::Response<stack_exchange::Answer> =
+                    response.json().await?;
+                let results = process_answers(answers)?;
 
-                let v = body.to_vec();
+                //
+                // NOTE:
+                // The "open" command is likely MacOS specific.
+                //
+                Command::new("open").args(results).output()?;
```

Comment:
It is mac specific, the command under most X windows linux systems is `xdg-open`. Probably the best way to capture this is to have a function for running open that takes just the `results`. and then calls the right command.  Or, you could make a sub-module with just the "open" command name as a const value.  That's a bit simpler actually.